examples/data-qa/sql-chat/utils.py

```diff
@@ -69,7 +69,9 @@ def get_database_uri() -> str:
 
     # Handle if no close match is found.
     if scheme == "No match found":
-        print(f"No close match found for '{scheme_input}'. Please verify your input.")
+        print(
+            f"No close match found for '{scheme_input}'. Please verify your" " input."
+        )
```

Comment:
these changes are coming because you seem to be using a different lint settings. I don't get these changes in my env.